### ISSUE TITLE
Button contrast not high enough

### ISSUE THREAD

**Issue by philli-m** [liqd/adhocracy-plus#2451]
**URL:** https://aplus-dev.liqd.net/ - light background black button
**URL:** https://aplus-dev.liqd.net/phillitopictest-org/dashboard/settings/ - language switch active not high enough contrast

If needed or helpful you can add a screenshot. Since issues are public, make sure to not display personal or secret data.

**Comment by TeaJayyy** [comment]
@sblindow Could you check whether this is still the case?
